src/test/script_tests.cpp

```diff
@@ -1744,10 +1746,11 @@ BOOST_AUTO_TEST_CASE(script_assets_test)
     BOOST_CHECK(tests.isArray());
     BOOST_CHECK(tests.size() > 0);
 
-    ELEMENTS: temporarily disabled until we implement the new Taproot sighash and upload new qa-assets
+    g_con_elementsmode = true;
```

Comment:
In feac5cf1d57be065d568bae69e756028a207054c:

Have you tested this? I am quite certain it is not going to work because the boost test harness is multithreaded.

Reply:
oh interesting, I'm certain the test itself worked, but I'll run `make -j X check` to test all and confirm

Reply:
Yes, the test will work by itself but dozens of other tests will fail.

Reply:
Ok, I'll try to reproduce locally. If it doesn't seem to cause problems then I guess it's okay.

Reply:
It fails for me. `*** 160 failures are detected in the test module "Bitcoin Core Test Suite"`.

Reply:
Actually, these failures all seem to be in the assets test itself ... I need to investigate further.

Reply:
maybe an obvious question: are you using the updated qa-assets from ElementsProject/qa-assets#1

Reply:
Derp, that was it. Let me merge that over there.